versioned/persist/serialize-proto/src/main/proto/persist.proto

```diff
@@ -60,12 +60,12 @@ message ContentId {
 
 message ContentIdWithType {
   ContentId content_id = 1;
-  int32 type = 2;
+  int32 type_unused = 2; // ignore the value of this field
```

Comment:
will this potentially cause issues after an upgrade?

Reply:
Nope. protobuf uses the int values - not the names